x/ante/ante.go

```diff
@@ -138,7 +138,7 @@ func checkMinimumFees(
 		return nil
 	}
 	// NB: if user pay insufficent fiat amount plus enough stake denom, fiat amount will be withdraw from the wallet anyway.
-
+	return nil
```

Comment:
This should be removed before merge. We need to discuss a solution to be able to use the stack with two chains to test ibc. For now, however, it is better to remove or comment